syntax = "proto3";

package autoxtime.proto;

message CarClass {
  optional int64 car_class_id = 1;
  optional string name = 2;
  optional int64 region_id = 3;
  optional string pax_formula = 4;
}
